src/app/containers/Article/index.jsx

```diff
@@ -26,14 +26,15 @@ const metadataProps = (amp, config, id, metadata, promo, service) => {
     amp,
     articleAuthor: config.articleAuthor,
     articleSection: metadata.passport.genre,
+    brandName: config.brandName,
```

Comment:
Rather than pass a new `brandName`, we can just rename `opengraphSiteName` to `brandName` and use that everywhere